### ISSUE TITLE
Anisotropic orders s.t. order 0 in some dim

### ISSUE THREAD

**Issue by santiagobadia** [gridap/Gridap.jl#306]
The generator of nodes that we have now in Gridap does not work in the case in which we have an anisotropic orders and the order is zero in one direction.

This is needed e.g. for spacetime formulations with Backward Euler integration in time.

**Comment by santiagobadia** [comment]
Working!

![cdg](https://user-images.githubusercontent.com/39598249/86326178-a9481580-bc84-11ea-87b2-a07af56aa369.png)

**Comment by santiagobadia** [comment]
In PR [#308 ](https://github.com/gridap/Gridap.jl/pull/308)

Closing the issue

**Comment by fverdugo** [comment]
Mola! 

Its very nice to look at the interpolation of the FESpace with a refined mesh. High order functions look really beautiful!

**Comment by fverdugo** [comment]
@santiagobadia I repoen the issue since it seems that we still cannot built Lagrangian reffes with 0 in one of the anisotropic orders

```julia
julia> reffe = LagrangianRefFE(Float64,QUAD,(2,0))
ERROR: AssertionError: If an order is 0 in some direction, it should be 0 also in the others
Stacktrace:
 [1] _compute_constant_nodes at /home/fverdugo/Code/jl/Gridap.jl/src/ReferenceFEs/LagrangianRefFEs.jl:584 [inlined]
 [2] _compute_nodes at /home/fverdugo/Code/jl/Gridap.jl/src/ReferenceFEs/LagrangianRefFEs.jl:575 [inlined]
 [3] compute_nodes at /home/fverdugo/Code/jl/Gridap.jl/src/ReferenceFEs/LagrangianRefFEs.jl:755 [inlined]
 [4] LagrangianRefFE(::Type{Float64}, ::ExtrusionPolytope{2}, ::Tuple{Int64,Int64}) at /home/fverdugo/Code/jl/Gridap.jl/src/ReferenceFEs/LagrangianRefFEs.jl:465
 [5] top-level scope at REPL[8]:1
```

**Comment by santiagobadia** [comment]
You are using a `LagrangianRefFE` with 0 order, that is not possible in `Gridap` (previous version before your changes).

For discontinuous spaces, I created `CDLagrangianRefFE`, in which I compute nodes with a modified algorithm.

So, one solution is to use the nodal construction in `CDLagrangianRefFE` in `LagrangianRefFE`, since it is more general

**Comment by santiagobadia** [comment]
@fverdugo I have worked on `CDLagrangianRefFE.jl` (and a minor modification in `LagrangianRefFE.jl`) in order to fix this problem,, see PR #312.

In any case, it is far from perfect. I don't like the fact that `LagrangianRefFE` constructor for some (but not all) orders equal to zero creates a `GenericRefFE` whereas it creates a `LagrangianRefFE` otherwise. And their first common ancestor is `ReferenceFE`.

We could probably think about this (internal) issue. One of the reasons could be the fact that we still keep dof ownership in `ReferenceFE`s. I would eliminate this concept from `ReferenceFE`s.

**Comment by fverdugo** [comment]
Closing issue, WIP in #314
